lms/djangoapps/teams/static/teams/js/views/topic_teams.js

```diff
@@ -25,21 +26,21 @@
                         this.collection.refresh(),
                         this.teamMemberships.refresh()
                     ).done(function() {
-                        TeamsView.prototype.render.call(self);
+                            TeamsView.prototype.render.call(self);
 
-                        if (self.teamMemberships.canUserCreateTeam()) {
-                            var message = interpolate_text(
-                                _.escape(gettext("Try {browse_span_start}browsing all teams{span_end} or {search_span_start}searching team descriptions{span_end}. If you still can't find a team to join, {create_span_start}create a new team in this topic{span_end}.")),
-                                {
-                                    'browse_span_start': '<a class="browse-teams" href="">',
-                                    'search_span_start': '<a class="search-teams" href="">',
-                                    'create_span_start': '<a class="create-team" href="">',
-                                    'span_end': '</a>'
-                                }
-                            );
-                            self.$el.append(_.template(teamActionsTemplate, {message: message}));
-                        }
-                    });
+                            if (self.teamMemberships.canUserCreateTeam()) {
+                                var message = interpolate_text(
+                                    _.escape(gettext("{browse_span_start}Browse teams in other topics{span_end} or {search_span_start}search team descriptions{span_end} in this topic. If you still can't find a team to join, {create_span_start}create a new team in this topic{span_end}.")),
```

Comment:
This is pretty complex for translators. Anything we can do to improve it? At the very least, could use a translator comment.